docs/core/install/sdk.md

```diff
@@ -128,7 +128,7 @@ While Visual Studio Code doesn't come with an automated .NET Core installer like
 
 The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the SDK. You can download the script from the [dotnet-install script reference page](../tools/dotnet-install-script.md).
 
-The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version, which is .NET Core 2.1. To install the current release of .NET Core, run the script with the following switch.
+The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version, which is .NET Core 3.1. To install the current release of .NET Core, run the script with the following switch.
```

Comment:
```suggestion
The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version. To install the current release of .NET Core, run the script with the following switch.
```

Reply:
I think we should keep the version. This makes it so that the user doesn't have to click on something to go find out what version would be installed, they are told upfront.